src/components/MenuBar/TimestampQueryBox.css

```diff
@@ -0,0 +1,24 @@
+.timestamp-query-box {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.timestamp-query-box-input {
+    box-sizing: border-box;
+    height: var(--ylv-menu-bar-height);
```

Comment:
shall we set the `height: var(--ylv-menu-bar-height)` on the `.timestamp-query-box` and set `height: 100%` here?